Engine/Fuego/Events/ApplicationEvent.h

```diff
@@ -13,11 +13,11 @@ class FUEGO_API WindowResizeEvent : public Event
     {
     }
 
-    inline unsigned int GetWidth() const
+    unsigned int GetWidth() const
     {
         return m_Width;
     }
-    inline unsigned int GetHeight() const
+    unsigned int GetHeight() const
     {
         return m_Height;
```

Comment:
Do not make implementation of non-template classes inside a header. Any implementation changes inside a header lead to recompilation of sources which depend on this header.

Reply:
Discussed in our chat